tests/factories/cht/reports/delivery.js

```diff
@@ -0,0 +1,260 @@
+const Factory = require('rosie').Factory;
+const uuid = require('uuid');
+const moment = require('moment');
+
+const defaultFields = {
+  'inputs': {
+    'meta': {
+      'location': {
+        'lat': '',
+        'long': '',
+        'error': '',
+        'message': ''
+      },
+      'deprecatedID': ''
+    },
+    'source': 'contact',
+    'source_id': '',
+    'user': {
+      'contact_id': '2e0ceb06-ced2-5a63-bca0-0283a5aab0e8',
+      'name': 'Andra Anderson',
+      'phone': '+9779841299392'
+    },
+    'contact': {
+      '_id': '091d10d9-2f5c-4965-9c20-96730c6fc300',
+      'name': 'James',
+      'short_name': '',
+      'patient_id': '96854',
+      'date_of_birth': '1997-11-18',
+      'sex': 'female',
+      'parent': {
+        '_id': '8935e9d8-0b2a-5eb0-ae54-ad1377a60097',
+        'parent': {
+          '_id': '2112ab72-448a-50bb-a15d-0b19d2927ab7',
+          'parent': {
+            '_id': ''
+          },
```

Comment:
This is a great place to use the factories in a factory... here we're having to make up parent ids, which won't likely match up to anything real. If using a factory you'll end up with real objects and useful ids throughout. 

This big object has a lot of data going on and if it's going to be a huge effort to rework it all, it's ok to handle this in a subsequent PR and make progress now without having this all be blocked for so long.